src/cxristian/Matriz_Cuadrada.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+
+public class Matriz_Cuadrada {
+    private Scanner keyboard;
+    private int[][] mat;
```

Comment:
It's recommendable to use names easy to understand for your variables/functions. What about 'matrix' instead?